src/routes/embedded/wallet/transactions-history/transactions-history.view.tsx

```diff
@@ -1,27 +1,33 @@
 import { Loading } from "@arconnect/components-rebrand";
-import { Card, Text, Box, Button } from "~components/embed/ui";
+import { Text, Box, Button } from "~components/embed/ui";
 import browser from "~iframe/browser";
 import { useActiveWallet, useTransactions } from "~wallets/hooks";
 import { useLocation } from "~wallets/router/router.utils";
 import TransactionGroup from "../transactions/components/TransactionGroup";
+import { AuthRequestCard } from "~components/embed/ui/molecules/card/auth-request-card/AuthRequestCard";
 
 export function WalletTransactionsHistoryEmbeddedView() {
   const { address } = useActiveWallet();
   const { navigate } = useLocation();
   const { transactions, loading, hasNextPage, fetchTransactions, count } = useTransactions(address);
 
   return (
-    <Card
-      size="auto"
+    <AuthRequestCard
```

Comment:
This card takes the full height of the screen if there are lot of transactions to render. Is this expected ?

Reply:
For now, yes. I'll create a new issue to adjust the max height on the SDK side (so that devs can change it if they want to).